spec/system/foods/new_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'food/new', type: :system do
+  let(:user) { create(:user) }
+
+  before do
+    sign_in(user)
+    user.confirm
+    visit new_food_path
+  end
+  describe 'This view have to render the correct fields' do
+    it 'renders the new food form' do
+      expect(page).to have_content('Add Food:')
+    end
```

Comment:
In this case, it would be good to be more specific with the selector because asserting over content doesn't necessarily mean that there's a form. An excellent way to approach this could be using the have_selector matcher that capybara provides and looking for the element `form` with the id `new-food-form` as you defined.

```rb
    it 'renders the new food form' do
      expect(page).to have_selector('form', id:'new-food-form')
    end
```